<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLvPopover</class>
 <widget class="QWidget" name="AddLvPopover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string>New Logical Volume</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="optionsWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="SizeRangeBox" name="sizeRangeBox" native="true">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="createButton">
        <property name="text">
         <string>Create Logical Volume</string>
        </property>
        <property name="icon">
         <iconset theme="list-add">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="tSubtitleLabel" name="label">
        <property name="text">
         <string>Logical Volume Options</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="formatBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tSubtitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">tsubtitlelabel.h</header>
  </customwidget>
  <customwidget>
   <class>SizeRangeBox</class>
   <extends>QWidget</extends>
   <header>components/sizerangebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
